build.psm1

```diff
@@ -907,12 +853,8 @@ function Start-PSPester {
     if ($Environment.IsWindows) {
         $Command += "Set-ExecutionPolicy -Scope Process Unrestricted; "
     }
-    $startParams = @{binDir=$binDir}
 
-    if(!$FullCLR)
-    {
-        $Command += "Import-Module '$Pester'; "
-    }
+    $Command += "Import-Module '$Pester'; "
```

Comment:
I think we should keep it as the `-Pester` parameter can be given a different path than the one in our module path.